dht.go

```diff
@@ -417,7 +412,23 @@ func (dht *IpfsDHT) Mode() ModeOpt {
 	return dht.auto
 }
 
-// fixLowPeersRoutine tries to get more peers into the routing table if we're below the threshold
+func (dht *IpfsDHT) populatePeers(_ goprocess.Process) {
+	if !dht.disableFixLowPeers {
+		dht.fixLowPeers(dht.ctx)
+	}
+
+	if err := dht.rtRefreshManager.Start(); err != nil {
+		logger.Error(err)
+	}
```

Comment:
Let's not put this in a separate goprocess and leave it in the init function, it's nice to return the errors on init since if we fail to start the refresh manager the DHT is dead. A log error, isn't nearly as good.

Reply:
This is contrary to what @petar said above:

> FixLowPeers should be called and finish strictly before RefreshManager starts

Reply:
Ah fair enough, for now this may to reduce start up errors from refreshing the routing table while it's still empty.

However, calling `host.Connect()` on a bootstrap peer will not actually add the peer to the routing table immediately as that's triggered by a separate event. If we want to reduce those errors we'll have to add something that actually waits on peers initially being added to the routing table.